### DISCUSSION TITLE
Refund does not change budget?

### DISCUSSION THREAD

**Discussion by stephanvierkant** [firefly-iii/firefly-iii#7707] [General]
Here is my budget of my insurance: (€152.33 a month)
![Screenshot_20230701_205241](https://github.com/firefly-iii/firefly-iii/assets/601833/1ad9ac64-53c8-485b-9a38-ea893e2593b6)

But my insurance company made a mistake and has done a partial refund. I've marked those transactions as a refund:
![Screenshot_20230701_205437](https://github.com/firefly-iii/firefly-iii/assets/601833/d137c7df-1045-4d27-8466-edc0c06a91d8)

Now I expect my budget to be (€152.33-€5.21=€147,12), but it doesn't take the refund into account. Am I doing something wrong, or is this expected behaviour?

**Comment by JC5** [comment]
This is expected behavior. You can choose for yourself what you do with your budget.

**Reply by stephanvierkant** [reply]
I don't understand your reply. Why is my used budget >150? It should take the refund into account imho.

**Reply by JC5** [reply]
It doesn't. That's how it works. 

https://docs.firefly-iii.org/firefly-iii/faq/financials/
